theme/SLE/wizard/installation.qss

```diff
@@ -484,7 +484,9 @@ QToolButton {
   border: 1px solid #969494;
   border-radius: 2px;
   background-color: #2B2E38;
-  padding: 8px 16px;
+/* Commenting this fixes the icons in the file selection dialog, */
+/* but where else is it used? Would it break something? */
```

Comment:
In fact, it was added recently by @kwwii in https://github.com/yast/yast-theme/commit/4471577599a4bc49dec6414baf911f348e646230. But it could be a leftover of some testing.

@kwwii, Do you remember something related to the padding added to a QToolButton? Thanks!